### DISCUSSION TITLE
ModuleNotFoundError: No module named 'torch_geometric.logging'

### DISCUSSION THREAD

**Discussion by 152005287879** [pyg-team/pytorch_geometric#5994] [Q&A]
![image](https://user-images.githubusercontent.com/45591569/202338354-7f617940-59f4-44d4-a3c7-b402cce47827.png)[](url)
excuse me!sir,how to solve above problem?
![image](https://user-images.githubusercontent.com/45591569/202338488-63b09f2a-78f1-4b9a-a407-47e91df2b1ca.png)
![image](https://user-images.githubusercontent.com/45591569/202338548-d70159a6-e5c3-4cf7-80f5-d7d6901e63d7.png)
![image](https://user-images.githubusercontent.com/45591569/202338591-2cc82e94-bb18-4891-bb06-a6885c74bbf3.png)
![image](https://user-images.githubusercontent.com/45591569/202338665-44832a7e-87c4-47b4-aa9a-bc3dc63e918d.png)
![image](https://user-images.githubusercontent.com/45591569/202338701-8a820281-bdc6-4f27-86b1-34ecdee567a6.png)

**Comment by rusty1s** [comment]
Which PyG version are you using? I can't find it in your screenshots :( You might need to upgrade to a later version. Alternatively, just remove the logging calls by jumping back to an earlier version, e.g., [this one](https://github.com/pyg-team/pytorch_geometric/blob/66b17806b1f4a2008e8be766064d9ef9a883ff03/examples/gcn.py).

**Comment by 152005287879** [comment]
Thank you for your reply，my PyG version is 2.1.0，but my pyg package version is 2.0.4,&nbsp; I took your advice&nbsp; and jump back to an earlier version:2.0.4,now ,I can run it! Thank you very much!


------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "pyg-team/pytorch_geometric"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年11月17日(星期四) 下午2:21
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [pyg-team/pytorch_geometric] ModuleNotFoundError: No module named 'torch_geometric.logging' (Discussion #5994)





 
Which PyG version are you using? I can't find it in your screenshots :( You might need to upgrade to a later version. Alternatively, just remove the logging calls by jumping back to an earlier version, e.g., this one.
